### DISCUSSION TITLE
client 4.9.5 消费不均衡

### DISCUSSION THREAD

**Discussion by zhengsh** [apache/rocketmq#8181] [General]
顺序消息， topic 有 128 个队列, 分 4 台机器去消费，server01 比其他机器的消息数量高2倍。

使用时 consumer 消费组消费多个 topic , 日志上来看，线程名一直在变。大佬们有类似的问题吗

**Comment by zhiliatom** [comment]
顺序消息会根据 shardingkey 对消息分片，观察brokeroffset位点，是否第一个 broker 的消息投递数量就是第二个 broker 的两倍呢？
